templates/default/sftd.yaml.erb

```diff
@@ -0,0 +1,85 @@
+---
+# Common Configuration Options:
+#
+# NOTE: This file managed by Chef.
+# Manual changes will be removed on next chef-client run
+#
+<% sft = node['scaleft'] %>
+
+<% if sft['access_address'] %>
+AccessAddress: <%= sft['access_address'] %>
+<% else %>
+# AccessAddress is unset by default
+<% end %>
+
+<% if sft['alt_names'] %>
+AltNames:  [<%= sft['alt_names'].join(",") %>]
+<% else %>
+# AltNames is unset by default
+<% end %>
+
+<% if sft['auto_enroll'] %>
```

Comment:
Since you set the default attribute to true you shouldn't need this logic. Looking at the documentation this matches the default of true too so looks good.